Readme.md

```diff
@@ -1,24 +1,16 @@
 # INTRODUCTION
-This will change life of your laptop's battery 
-This app will notify the user to remove the charger when his battery will be fully charged or is about to die.
+This will improve your laptop’s battery life.
+This app will notify the user to unplug the charger when his/her battery is fully charged or about to die.
```

Comment:
@RonIce  isn't better `his/her laptop's battery `

Reply:
@daudmalik06 ..I think the first sentence about improving the laptop's battery life carries through to the second sentence, that the need to include the word laptop on the second sentence become redundant.

What's your thoughts on that

Reply:
@daudmalik06 ..I think the first sentence about improving the laptop's battery life carries through to the second sentence, that the need to include the word laptop on the second sentence become redundant.

Do you prefer i explicitly put the word laptop on the sentence?

Reply:
i think yeah option i suggested is more readable
if that's possible can you do that ?
thanks @RonIce

Reply:
@daudmalik06..No problem, I'll add it now